ui/tempo-plugin/src/model/api-types.ts

```diff
@@ -65,10 +65,10 @@ export interface TraceSearchResponse {
 
 export interface SpanSearchResponse {
   spanID: string;
-  name: string;
+  name?: string;
```

Comment:
maybe the associated cuelang file should be updated as well.

Reply:
There are no cuelang files for the response from the Tempo API

Reply:
oh right sorry about that 😓